src/api/java/com/minecolonies/api/colony/colonyEvents/descriptions/NBTTags.java

```diff
@@ -0,0 +1,13 @@
+package com.minecolonies.api.colony.colonyEvents.descriptions;
+
+/**
+ * NBT tags used in colony events.
+ */
+public abstract class NBTTags
```

Comment:
Use the `NBTTagConstantsClass`

Reply:
ok, i did it like this because that is how `IColonyEvent` handles its tags.

Reply:
Then fix that too :smile:

Reply:
done.